.github/workflows/ci.yml

```diff
@@ -0,0 +1,14 @@
+name: ci
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+jobs:
+  build-linux:
+    runs-on: ubuntu-latest
+    container: swiftlang/swift:nightly-main-jammy
+    steps:
+      - uses: actions/checkout@v4
+      - run: apt-get update && apt-get install --no-install-recommends -y make llvm
```

Comment:
`apt-get install llvm` is a problem. It turned out that it overwrited Swift's builtin LLVM tools, so LLVM had been downgraded from 17 to 14.